src/Moryx.Products.Management/Modification/ProductConverter.cs

```diff
@@ -131,7 +142,7 @@ private PartModel ConvertPart(IProductPartLink link)
             var part = new PartModel
             {
                 Id = link.Id,
-                Product = ConvertProduct(link.Product, true),
+                Product = link.Product is null ? null : ConvertProduct(link.Product, true),
```

Comment:
We should not have part links without products. Even if that happens, we shouldn't return it like that from the API.

Reply:
But as the ProductConverter needs to handle custom Product(PartLink)s as well, shouldn't we cover this scenario?

Reply:
Resolved. We follow the convention that ProductPartLinks without attached Products have no use and, therefore, don't need to be cared for in the UI. The current changes take care of filtering such PartLinks if they occur anyway.